site/content/docs/master/plugins.md

```diff
@@ -166,6 +166,19 @@ By providing the flag `--show-default-podspec` to `sonobuoy gen`, the default `p
 If you generate a new manifest by running `sonobuoy gen` again, you will need to reapply any changes made.
 We recommend adding your desired customizations to the plugin definition itself.
 
+#### Plugin Installation (Experimental Feature)
+
+When you select which plugins to run, currently you have to specify a local file or a URL where the plugin definition exists. Keeping track of the absolute paths and URLs creates a burden to plugin adoption so we've implemented a new feature to help: plugin installation.
+
+To (enable)[featureGates], set `SONOBUOY_PLUGIN_INSTALLATION=true`. Then proceed to use Sonobuoy normally. The new functionality is that you can install a plugin for repeated use by executing the commands:
```

Comment:
That last sentence here sounds like it's missing something.
Maybe `With this new functionality...`